src/main/java/com/server/bugtracker/User/User.java

```diff
@@ -16,8 +16,8 @@
     private long id;
     @Column(name = "name")
     private String name;
-    @Column(name = "user_name")
```

Comment:
Your name change ("username") will not work with the current database set up. Please remove your changes or explain why the change was made.

Reply:
the variable user_name doesnt follow java naming convention so the repo throws me an error. 
the column annotation can revert back to user_name 
https://stackoverflow.com/questions/23456197/spring-data-jpa-repository-underscore-on-entity-column-name

Reply:
You're correct. 

The variable name can be changed but the annotated column cannot be changed.

Reply:
I'd say go ahead and change the variable names (getters/setters and constructor) to match Java naming conventions. However, please keep in mind that the annotated column name MUST stay as-is (underscore).

Thanks.